appcli/cli_builder.py

```diff
@@ -125,6 +125,11 @@ def create_cli(configuration: Configuration, desired_environment: Dict[str, str]
         multiple=True,
         callback=extract_valid_environment_variable_names,
     )
+    @click.option(
+        "--non-tty",
+        help="Exclude tty specific commands from the launcher script.",
```

Comment:
The expected user of `appcli` is likely a sysadmin, who may understand what a tty is, but not why we'd want to exclude tty from commands in the launcher script.
I think user-facing comment is just a little too deep in the weeds, and it may be worth explaining 'why' you would want to exclude tty.

`tty specific commands` doesn't quite make sense in this context either, it's more that the launcher is not attaching a tty during command evaluation, which is using when you don't have a terminal (e.g. cron entries)